backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidInfo.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.backends.android;
+
+/** @author sarkanyi */
+public class AndroidInfo {
+	public static ZipResourceFile expansionFile;
```

Comment:
I'd prefer AndroidFiles to hold the reference to the expansion file, and pass it to the AndroidZipFileHandle constructor. If you need to add a configuration argument or method to set the expansion file on AndroidFiles, that's probably better than holding it in a public static.